### PR TITLE
Add installation instructions for RabbitMQ to the documentation

### LINKED ISSUE #1059: Documentation for RMQ installation
Documentation needs to be added when installing AiiDA to tell users how to install and setup rabbitMQ.

### PR DESCRIPTION
**Pull request by sphuber**
Fixes #1059

### CODE REVIEW

File: docs/source/install/details/rabbitmq.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+	sudo port install rabbitmq-server
+	sudo launchctl load -w /Library/LaunchDaemons/org.macports.rabbitmq-server.plist
+
+
+Gentoo
+------
+
+
+To install RabbitMQ on a Gentoo distribution through the ``portage`` package manager run the following command:
```
Comment (giovannipizzi) [inline docs/source/install/details/rabbitmq.rst]: The first part of this section should go in the other file (os.rst)

File: docs/source/install/details/rabbitmq.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+------
+
+After installing RabbitMQ, potentially with a system reboot, it should be started automatically after every system boot.
+You can check whether it is running by checking the status through the command:
+
+.. code-block:: bash
+
+	sudo rabbitmqctl status
+
+If you are having problems installing RabbitMQ, please refer to the detailed instructions  provided on the `website of RabbitMQ itselffor Debian based distributions <https://www.rabbitmq.com/install-debian.html>`_.
```
Comment (giovannipizzi) [inline docs/source/install/details/rabbitmq.rst]: typo, missing space 'itselffor'

File: docs/source/install/installation.rst [hunk 3]
```diff
@@ -7,8 +7,8 @@ Installation
 This section of the manual will guide you through the process of installing AiiDA on your system.
 AiiDA has been tested to run on the following platforms:
 
+* Ubuntu 14.04
```
Comment (giovannipizzi) [inline docs/source/install/installation.rst]: Isn't it working anymore on 16.04?

Comment (sphuber) [reply docs/source/install/installation.rst]: No it is, but below it says that it works for other Debian based distributions so I though that covered it. But I will put the explicit reference back for 16.04

**Pull request merged**